bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -162,6 +172,11 @@ spec:
           name: run-openvswitch
         - mountPath: /env
           name: env-overrides
+        env:
```

Comment:
@JacobTanenbaum don't need these env vars for this container.

Reply:
done